locations/spiders/nandos_za.py

```diff
@@ -13,26 +13,33 @@ class NandosZASpider(scrapy.Spider):
     start_urls = [
         "https://api.locationbank.net/storelocator/StoreLocatorAPI?clientId=67b9c5e4-6ddf-4856-b3c0-cf27cfe53255"
     ]
+    web_root = "https://store.nandos.co.za/details/"
 
     def parse(self, response):
         data = response.json()
         for i in data["locations"]:
             i["name"] = i.pop("locationName")
             i["phone"] = i.pop("primaryPhone")
+            if i["additionalPhone1"]:
+                i["phone"] += ";" + i.pop("additionalPhone1")
```

Comment:
Actually, is this a good idea?
It gives results like `"phone": "010 216 9143;010 216 9149"`, rather than `"phone": "+27 10 216 9143"`

Reply:
Ping @iandees I was just too late commenting it seems

Reply:
```suggestion
                i["phone"] += "; " + i.pop("additionalPhone1")
```

Reply:
Or rather allow it not to have a space, which is common in OSM and seems just as valid. See #9548